presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkTaskInputs.java

```diff
@@ -27,18 +27,18 @@
 public class PrestoSparkTaskInputs
 {
     // fragmentId -> Iterator<[partitionId, page]>
-    private final Map<String, Iterator<Tuple2<Integer, PrestoSparkRow>>> shuffleInputs;
+    private final Map<String, Iterator<Tuple2<MutablePartitionId, PrestoSparkRow>>> shuffleInputs;
```

Comment:
actually use `MutablePartitionId` instead of `Integer` make generic type parameters slightly easier to understand ;)